app/services/constituent_service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+# TODO: this should move to dor-services-app and dor-services-client
```

Comment:
Moving to dor-services-app: https://github.com/sul-dlss/dor-services-app/pull/321